build failure due to typo in junit file path

**Describe the issue**: please provide a clear and concise description of what the bug is.  

build failed, junit failure.
See https://gitlab.com/pc2ccs/pc2v9/-/jobs/2342033859 for details.

**To Reproduce**: please describe, in order, the steps to reproduce the behavior:

merge/push to i_295_develop_work

**Expected behavior**: please provide a description of what you *expected* to happen when you took the above steps:  

clean build, no errors

**Actual behavior**: please describe what _actually_ happened when you took the above steps:

Testcase: testLoadSandboxAndMemoryLimit(edu.csus.ecs.pc2.imports.ccs.ContestSnakeYAMLLoaderTest):	Caused an ERROR
    [junit] Problem loading samps/contests/SumitMTC/config/contest.yaml
edu.csus.ecs.pc2.core.exception.YamlLoadException: Problem loading samps/contests/SumitMTC/config/contest.yaml
Caused by: java.io.FileNotFoundException: samps/contests/SumitMTC/config/contest.yaml

**Environment**: please complete the following information:

gitLab

**Log Info**:  PC^2 generates logs (in the **logs** folder for client modules; under the **profiles** folder for the PC2 Server).  

**Screenshots**:  if applicable, please add attachments showing screenshots to help explain the issue.

**Additional context**: please add any other context about the issue here.

--- a/test/edu/csus/ecs/pc2/imports/ccs/ContestSnakeYAMLLoaderTest.java
+++ b/test/edu/csus/ecs/pc2/imports/ccs/ContestSnakeYAMLLoaderTest.java
@@ -3679,7 +3679,7 @@ public class ContestSnakeYAMLLoaderTest extends AbstractTestCase {
         /**
          * Contest to use as input
          */
-        String sampleContestDirName = "SumitMTC";
+        String sampleContestDirName = "sumitMTC";
         
         IInternalContest contest = new InternalContest();
         

